tests/CMakeLists.txt

```
@@ -342,10 +342,9 @@ set_tests_properties(FilteredSection::GeneratorsDontCauseInfiniteLoop-2
     PASS_REGULAR_EXPRESSION "All tests passed \\(4 assertions in 1 test case\\)"
 )
 
-# AppVeyor has a Python 2.7 in path, but doesn't have .py files as autorunnable
 add_test(NAME ApprovalTests
   COMMAND
-    ${PYTHON_EXECUTABLE}
+    "${Python3_EXECUTABLE}"
```

```suggestion
    Python3::Interpreter
```
Can we use the `IMPORTED` `Interpreter` target that FindPython3.cmake provides?

https://cmake.org/cmake/help/v3.16/module/FindPython3.html#imported-targets

---

Replaced with imported target.